### DISCUSSION TITLE
Release/5.3.3

### DISCUSSION THREAD

**Discussion by Athanasius** [EDCD/EDMarketConnector#1506] [Announcement]
Release 5.3.3
===

Unfortunately 5.3.2 failed to fully address the issues caused by the different Journal filenames when using the Odyssey Update 11 client.  It's fine if you run EDMarketConnector first and *then* the game, as the code path that detects a new file always does just that.

But the code for EDMarketConnector startup to find the current newest Journal file relied on sorting the filenames and that would mean the new-style names would always sort as 'oldest'.

This release fixes that code to properly use the file modification timestamp to determine the newest file on startup.

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.3.3'>Release/5.3.3</a>.</em>

**Comment by judgechuck** [comment]
Hi Athanasius, just wanted to thank you for the quick response and fix. All working fine, as I am sure you are already aware.

Regards,

JC
